config/software/server-jre.rb

```diff
@@ -15,8 +15,20 @@
 #
 
 name "server-jre"
-default_version "8u31"
-raise "Server-jre can only be installed on x86_64 systems." unless _64_bit?
+
```

Comment:
We need to break out the IBM specific stuff into its own software definition. I would call it `ibm-jre`.